templates/docker-entrypoint.ejs

```diff
@@ -13,6 +13,23 @@ const { writeFileSync } = require('node:fs')
 <% } -%>
 <% } -%>
 
+const env = {...process.env}
+
+<% if (adonisjs) { -%>
+if (process.env.DATABASE_URL) {
+<%= tab(1) %>try {
+<%= tab(2) %>const databaseUrl = new URL(process.env.DATABASE_URL)
+<%= tab(2) %>env.PG_HOST = databaseUrl.hostname
+<%= tab(2) %>env.PG_PORT = databaseUrl.port
+<%= tab(2) %>env.PG_USER = databaseUrl.username
+<%= tab(2) %>env.PG_PASSWORD = databaseUrl.password
+<%= tab(2) %>env.PG_DB_NAME = databaseUrl.pathname.slice(1)
+<%= tab(1) %>} catch (err) {
+<%= tab(2) %>console.error('Invalid DATABASE_URL')
+<%= tab(1) %>}
+}
+<% } -%>
```

Comment:
Move the next (blank) line into this if statement.  Otherwise non-adonisjs frameworks will get an extra blank line.